scripts/test-ci-all.sh

```diff
@@ -87,6 +87,8 @@ export -f cli_test_always_success
 tmpdir=$(mktemp --tmpdir -d XXXXX)
 trap 'rm -r $tmpdir' EXIT
 joblog="$tmpdir/joblog"
+trap 'if [ "$?" -ne 0 ]; then echo "##Client Database"; dbtool client list; echo "##Server Database"; dbtool server-0 list; fi' EXIT
```

Comment:
Does this work? I thought a path to the DB folder would be needed.

Reply:
Hello @m1sterc001guy, should I add `$FM_DATA_DIR/server-0/database`

Reply:
Give it a try in tmuxinator. I don't remember the full command, but that looks correct for dumping server-0's db.
Something like this:
`dbtool $FM_DATA_DIR/server-0/database dump -- $FM_DATA_DIR/server-0 pass0`

Reply:
Hello @m1sterc001guy, but in the issue it is mentioned to call `dbtool <db> list`. Or should I go ahead with your suggestion.